config/locales/simple_form.en.yml

```diff
@@ -51,6 +51,7 @@ en:
         severity: Severity
         type: Import type
         username: Username
+        username_or_email: Username or Email
```

Comment:
I don't see which form uses this label but it's again my concern about mixing PAM-users and registration-users.

Reply:
you have to uncomment config.pam_authentication = true in initializers.
This feature is optional.

Reply:
the login form uses it. It may wasn't there because of indentation issues.

Reply:
everything ok now? Or shall I change something?